scripts/coe_player.lua

```diff
@@ -4,6 +4,7 @@ local Player = {}
 local Config = require("config")
 local Log = require("utils/log")
 local Utils = require("utils/utils.lua")
+local Stats = require("scripts/coe_stats_gui")
```

Comment:
Player is a helper class, it shouldn't require anything. coe_stats_gui needs an onPlayerCreated event and have it call the setupStatsGui function

Then add StatsGui.OnPlayerCreated() to the event in control.lua